docker-compose.yml

```diff
@@ -1,53 +1,68 @@
-version: "2"
+version: '3'
```

Comment:
Which docker-compose directive requires v3 format? If we upgrade the compose files to v3, it will break backward compatibility. Not sure if it is good time to upgrade to v3 and if we decide to go with v3, we need to list it as a breaking change.